api.planx.uk/send/email.ts

```diff
@@ -44,9 +44,21 @@ const sendToEmail = async(req: Request, res: Response, next: NextFunction) => {
 
       // Send the email
       const response = await sendEmail("submit", settings.sendToEmail, config);
-      return res.json(response);
+      if (response?.message === "Success") {
+        // Mark session as submitted so that reminder and expiry emails are not triggered
+        markSessionAsSubmitted(payload.sessionId);
 
-      // TODO Mark lowcal_session as submitted, create/update audit table (and setup Slack notification trigger?)
+        // TODO create audit table entry? setup event trigger for slack notification on new row?
```

Comment:
Keeping this TODO comment around so we don't lose track of it, but no urgency for this step in my mind ! 

We have Hasura Event logs for auditing purposes in the meantime and can decide later if a Slack notification makes sense for discretionary email-only submission services (theoretically there isn't the same visibility concern here as Uniform connector for example).

Reply:
Yeah this seems fine to me also - we can address this as and when it's required.